open_event/static/js/jquery/jquery.smartWizard.js

```diff
@@ -0,0 +1,489 @@
+                    return false;
+                }
+            }
+        }
+        //$this.elmStepContainer.height(_step($this, selStep).outerHeight());
+        var prevCurStepIdx = $this.curStepIdx;
+        $this.curStepIdx =  stepIdx;
+        if ($this.options.transitionEffect == 'slide'){
+            _step($this, curStep).slideUp("fast",function(e){
+                _step($this, selStep).slideDown("fast");
+                _setupStep($this,curStep,selStep);
+            });
+        } else if ($this.options.transitionEffect == 'fade'){
+            _step($this, curStep).fadeOut("fast",function(e){
+                _step($this, selStep).fadeIn("fast");
+                _setupStep($this,curStep,selStep);
+            });
+        } else if ($this.options.transitionEffect == 'slideleft'){
+            var nextElmLeft = 0;
+            var nextElmLeft1 = null;
+            var nextElmLeft = null;
+            var curElementLeft = 0;
+            if(stepIdx > prevCurStepIdx){
+                nextElmLeft1 = $this.elmStepContainer.width() + 10;
+                nextElmLeft2 = 0;
+                curElementLeft = 0 - _step($this, curStep).outerWidth();
+            } else {
+                nextElmLeft1 = 0 - _step($this, selStep).outerWidth() + 20;
+                nextElmLeft2 = 0;
+                curElementLeft = 10 + _step($this, curStep).outerWidth();
+            }
+            if (stepIdx == prevCurStepIdx) {
+                nextElmLeft1 = $($(selStep, $this.target).attr("href"), $this.target).outerWidth() + 20;
+                nextElmLeft2 = 0;
+                curElementLeft = 0 - $($(curStep, $this.target).attr("href"), $this.target).outerWidth();
+            } else {
+                $($(curStep, $this.target).attr("href"), $this.target).animate({left:curElementLeft},"fast",function(e){
+                    $($(curStep, $this.target).attr("href"), $this.target).hide();
+                });
+            }
+
+            _step($this, selStep).css("left",nextElmLeft1).show().animate({left:nextElmLeft2},"fast",function(e){
+                _setupStep($this,curStep,selStep);
+            });
+        } else {
+            _step($this, curStep).hide();
+            _step($this, selStep).show();
+            _setupStep($this,curStep,selStep);
+        }
+        return true;
+    };
+
+    var _setupStep = function($this, curStep, selStep) {
+        $(curStep, $this.target).removeClass("selected");
+        $(curStep, $this.target).addClass("done");
+
+        $(selStep, $this.target).removeClass("disabled");
+        $(selStep, $this.target).removeClass("done");
+        $(selStep, $this.target).addClass("selected");
+
+        $(selStep, $this.target).attr("isDone",1);
+
+        _adjustButton($this);
+
+        if($.isFunction($this.options.onShowStep)) {
+            var context = { fromStep: parseInt($(curStep).attr('rel')), toStep: parseInt($(selStep).attr('rel')) };
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You should specify the radix of the mentioned string in parseInt calls.](https://www.codacy.com/app/triplez/open-event-orga-server/file/2668912768/issues/source?bid=3233864&fileBranchId=3324174#l273)